pkg/tsdb/opentsdb/opentsdb.go

```diff
@@ -424,3 +449,26 @@ func (s *Service) getDSInfo(ctx context.Context, pluginCtx backend.PluginContext
 
 	return instance, nil
 }
+
+func formatDownsampleInterval(ms int64) string {
```

Comment:
nit: should this function be in its own `utils` file? I think that a few functions after `createRequest` can be in the `utils` file

we can also maybe use go's `time` constants here to make it a bit cleaner and easier to understand